src/components/application_manager/test/commands/hmi/simple_requests_to_hmi_test.cc

```diff
@@ -138,10 +137,9 @@ typedef Types<commands::VIIsReadyRequest,
               commands::VIDiagnosticMessageRequest,
               commands::VIGetDTCsRequest,
               commands::VIGetVehicleDataRequest,
-              commands::VIUnsubscribeVehicleDataRequest,
-              commands::VRAddCommandRequest,
-              commands::VRChangeRegistrationRequest,
-              commands::VRDeleteCommandRequest> RequestCommandsList;
+              commands::UISetMediaClockTimerRequest,
+              commands::UIShowRequest,
+              commands::UISliderRequest> RequestCommandsList;
```

Comment:
@wolfylambova22, why do you exclude `VIUnsubscribeVehicleDataRequest`, `VRAddCommandRequest`, `VRChangeRegistrationRequest`, `VRDeleteCommandRequest`?